unlock-app/src/__tests__/middlewares/currencyConversionMiddleware.test.js

```diff
@@ -1,19 +1,16 @@
-import mockAxios from 'jest-mock-axios'
```

Comment:
Please note to get rid of that dep if we're not using it!

Reply:
noted, but I do believe this is being used elsewhere in the code base